services/core-web/src/components/noticeOfWork/applications/review/ReviewNOWApplication.js

```diff
@@ -1,5 +1,7 @@
+/* eslint-disable */
+
```

Comment:
disabled the entire file, the props are not being read correctly when passed into the "Field" component